model/file_info.go

```diff
@@ -15,6 +15,27 @@ import (
 	"strings"
 )
 
+const (
+	FILEINFO_SORT_BY_CREATED = "CreateAt"
+	FILEINFO_SORT_BY_SIZE    = "Size"
+)
+
+// GetFileInfosOptions contains options for getting FileInfos
+type GetFileInfosOptions struct {
+	// UserIds optionally limits the FileInfos to those created by the given users.
+	UserIds []string `json:"user_ids"`
+	// ChannelIds optionally limits the FileInfos to those created in the given channels.
+	ChannelIds []string `json:"channel_ids"`
+	// Since optionally limits FileInfos to those created after the given time, specified as Unix time in milliseconds.
```

Comment:
minor tweak to reflect code change:
```suggestion
	// Since optionally limits FileInfos to those created at or after the given time, specified as Unix time in milliseconds.
```

Reply:
Good catch, I completely missed this change when I updated the code.